dpkg/distroless_scanner.go

```diff
@@ -0,0 +1,145 @@
+package dpkg
+
+import (
+	"bufio"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"net/textproto"
+	"path/filepath"
+	"runtime/trace"
+
+	"github.com/quay/zlog"
+
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/indexer"
+	"github.com/quay/claircore/pkg/tarfs"
+)
+
+const (
+	distrolessName    = "dpkg-distroless"
+	distrolessKind    = "package"
+	distrolessVersion = "1"
+)
+
+var (
+	_ indexer.VersionedScanner = (*Scanner)(nil)
+	_ indexer.PackageScanner   = (*Scanner)(nil)
+)
+
+// Scanner implements the scanner.PackageScanner interface.
+//
+// This looks for directories that look like dpkg databases and examines the
+// "base" and "netbase" files it finds there.
```

Comment:
Only these two? `gcr.io/distroless/python3-debian11` has `openssl`, `libssl1`, and `libc6`

Reply:
Bad comment, updating